### ISSUE TITLE
Laravel elexir gulp compiler error

### ISSUE THREAD

**Issue by itrea** [n33/skel#150]
I get the following error when trying to compile skel with laravels elixir.
```
[10:08:05] gulp-notify: [Laravel Elixir] Sass Compilation Failed!: resources/assets/sass/skel.scss
  203:22  invalid selector after 25
{ [Error: resources/assets/sass/skel.scss
  203:22  invalid selector after 25]
  message: 'resources/assets/sass/skel.scss\n  203:22  invalid selector after 25',
  column: 22,
  line: 203,
  file: 'stdin',
  status: 1,
  messageFormatted: '\u001b[4mresources/assets/sass/skel.scss\u001b[24m\n\u001b[90m  203:22\u001b[39m  invalid selector after 25',
  name: 'Error',
  stack: 'Error: resources/assets/sass/skel.scss\n  203:22  invalid selector after 25\n    at options.error (/Applications/MAMP/htdocs/github/api.sipcom.dk/node_modules/gulp-sass/node_modules/node-sass/lib/index.js:276:32)',
  showStack: false,
  showProperties: true,
  plugin: 'gulp-sass' }
```

The lines that are refered to are
```sass
203     // Modifiers.
204       .container.\31 25\25    { width: 100%; max-width: $width * 1.25; min-width: $width; }
205       .container.\37 5\25     { width: $width * 0.75; }
206       .container.\35 0\25     { width: $width * 0.5; }
207       .container.\32 5\25     { width: $width * 0.25; }
```

**Comment by itrea** [comment]
I see this issue is related to laravel an not skel, Laravel uses libsass wich does not support the syntax. I have fixed my issue by using https://www.npmjs.com/package/laravel-elixir-ruby-sass in my project

**Comment by n33** [comment]
Yup, it's a libsass issue (which should be fixed in libsass 3.3; see https://github.com/n33/skel/issues/142)
